dataprofiler/profilers/profile_builder.py

```diff
@@ -1163,6 +1163,11 @@ def __init__(self, data, samples_per_update=None, min_true_samples=0,
         self._profile = []
         self._col_name_to_idx = defaultdict(list)
         self.correlation_matrix = None
+        self._corr_prev_properties = {
```

Comment:
if we decide to use this, should it be saved with the profiler so that later on we will load it and potentially merge with other profile?